docs/api/models/ollama.md

```diff
@@ -35,6 +35,32 @@ print(result.cost())
 #> Cost(request_tokens=56, response_tokens=8, total_tokens=64, details=None)
 ```
 
+## Example using a remove server
+
+```py title="ollama_example_with_remote_server.py"
+from pydantic import BaseModel
+from pydantic_ai import Agent
+from pydantic_ai.models.ollama import OllamaModel
+
+ollama_model = OllamaModel(
+    model_name='qwen2.5-coder:7b', # the name of the model running on your remote server
```

Comment:
might be better to use [code annotations](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations) like we do elsewhere.

Reply:
Yes nice, will update